### PR TITLE
added alt text

### PR DESCRIPTION
**Pull request by brynnelliott**
Added alt text contributed by Toby Hodges for https://swcarpentry.github.io/r-novice-gapminder//fig/rmd-08-lm-fit-1.png, https://swcarpentry.github.io/r-novice-gapminder//fig/rmd-08-lm-fit2-1.png, https://swcarpentry.github.io/r-novice-gapminder//fig/rmd-08-ch4a-sol-1.png, https://swcarpentry.github.io/r-novice-gapminder//fig/rmd-08-ch3-sol-1.png

### CODE REVIEW

**Comment by naupaka** [comment]
Thanks for the PR! In all cases this should be `fig.alt` not `fig. alt`

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 1]
```diff
@@ -194,7 +194,7 @@ lines.
 > >
 > > The lines now get drawn over the points!
 > >
-> > ```{r ch3-sol}
+> > ```{r ch3-sol, fig. alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The plot illustrates the possibilities for styling visualisations in ggplot2 with data points enlarged, coloured orange, and displayed without transparency."}
```
Comment (brynnelliott) [inline _episodes_rmd/08-plot-ggplot2.Rmd:197]:
```suggestion
> > ```{r ch3-sol, fig.alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The plot illustrates the possibilities for styling visualisations in ggplot2 with data points enlarged, coloured orange, and displayed without transparency."}
```

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 2]
```diff
@@ -235,15 +235,15 @@ The `scale_x_log10` function applied a transformation to the coordinate system o
 We can fit a simple relationship to the data by adding another layer,
 `geom_smooth`:
 
-```{r lm-fit}
+```{r lm-fit, fig. alt = "Scatter plot of life expectancy vs GDP per capita with a blue trend line summarising the relationship between variables, and gray shaded area indicating 95% confidence intervals for that trend line."}
```
Comment (brynnelliott) [inline _episodes_rmd/08-plot-ggplot2.Rmd:238]:
```suggestion
```{r lm-fit, fig.alt = "Scatter plot of life expectancy vs GDP per capita with a blue trend line summarising the relationship between variables, and gray shaded area indicating 95% confidence intervals for that trend line."}
```

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 3]
```diff
@@ -235,15 +235,15 @@ The `scale_x_log10` function applied a transformation to the coordinate system o
...
 ggplot(data = gapminder, mapping = aes(x = gdpPercap, y = lifeExp)) +
   geom_point(alpha = 0.5) + scale_x_log10() + geom_smooth(method="lm")
 ```
 
 We can make the line thicker by *setting* the **size** aesthetic in the
 `geom_smooth` layer:
 
-```{r lm-fit2}
+```{r lm-fit2, fig. alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The blue trend line is slightly thicker than in the previous figure."}
```
Comment (brynnelliott) [inline _episodes_rmd/08-plot-ggplot2.Rmd:246]:
```suggestion
```{r lm-fit2, fig.alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The blue trend line is slightly thicker than in the previous figure."}
```

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 4]
```diff
@@ -267,7 +267,7 @@ variables and their visual representation.
 > > This means that it applies to all data points on the graph and is not related to
 > > a specific variable.
 > >
-> > ```{r ch4a-sol}
+> > ```{r ch4a-sol, fig. alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The plot illustrates the possibilities for styling visualisations in ggplot2 with data points enlarged, coloured orange, and displayed without transparency."}
```
Comment (brynnelliott) [inline _episodes_rmd/08-plot-ggplot2.Rmd:270]:
```suggestion
> > ```{r ch4a-sol, fig.alt = "Scatter plot of life expectancy vs GDP per capita with a trend line summarising the relationship between variables. The plot illustrates the possibilities for styling visualisations in ggplot2 with data points enlarged, coloured orange, and displayed without transparency."}
```

**Comment by brynnelliott** [comment]
Thank you @naupaka! I think I have them fixed now.

**Comment by naupaka** [comment]
One of your changes was a 'suggested change', which appears to be some new GitHub feature (was news to me). I committed it so it will be included in the PR.

**Comment by naupaka** [comment]
Once the tests pass, this should be good to merge.

**Comment by naupaka** [comment]
@brynnelliott thanks for the contribution!

**Pull request merged**